CHANGELOG.md

```diff
@@ -7,6 +7,12 @@ Inspired from [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 ### Deprecated
 ### Removed
 ### Fixed
+- Fixed the response body being force-encoded when it was already in UTF-8. This improves compatibility with `webmock` >= 3.19 when stubbing a request without an explicitly defined response body.
```

Comment:
Sorry to be a pest, don't hate me ;) These CHANGELOGs have a specific 1-line format with a link to the GitHub issue or PR. Please drop the explanation and include that number, so if anyone wants more detail on the why they know where to look.

Reply:
Should be good now. I should have read through the contributing guidelines again before doing this, my bad really.

Reply:
No stress and THANK YOU for contributing!